### PR TITLE
Setup a github action to run tests

### PR DESCRIPTION
**Pull request by pattertom**
Run a test in each of the 4 languages to ensure changes to each solution don't break anything

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 2.5
+          bundler-cache: true
+          working-directory: ruby
+      - run: bundle exec rspec -e "does not include charges that only occur once" --format RSpec::Github::Formatter
+        working-directory: ruby
```
Comment (nc-thomaez) [inline .github/workflows/tests.yml:16]: Could set the working directory as a [default run option for the job](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_iddefaultsrun).

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    name: Javascript Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+          cache: 'npm'
+          cache-dependency-path: javascript/package-lock.json
+      - run: npm install && npm test -i test/Healthcheck.test.js
```
Comment (nc-thomaez) [inline .github/workflows/tests.yml:27]: Any particular reason not to split this into two steps?

File: java/gradlew [hunk 3]
```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
+#!/bin/sh
 
 #
-# Copyright 2015 the original author or authors.
+# Copyright © 2015-2021 the original authors.
```
Comment (nc-thomaez) [inline java/gradlew:4]: Should the license for this script be mentioned somewhere in the repo?

File: .github/workflows/tests.yml (hunk 2 above)
Comment (pattertom) [reply .github/workflows/tests.yml:27]: Just that I didn't know about default run option for a job, so I can update that

File: .github/workflows/tests.yml (hunk 1 above)
Comment (pattertom) [reply .github/workflows/tests.yml:16]: Why would that help? I think you still need to specify the directory for setup-ruby because it isn't a run step

Comment (nc-thomaez) [reply .github/workflows/tests.yml:16]: I think you're correct. Reading is fundamental.

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+      - run: bundle exec rspec -e "does not include charges that only occur once" --format RSpec::Github::Formatter
+        working-directory: ruby
+  javascript_tests:
+    name: Javascript Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '14'
```
Comment (nc-kreiserm) [inline .github/workflows/tests.yml:24]: `12`

**Review by nc-kreiserm — APPROVED** [review]
🌊 once rebased

**Pull request merged**